src/index.html

```diff
@@ -0,0 +1,133 @@
+                        Azawi - Slow Dancing <br>(Official Music Video)
+                    </p>
+                    <p class="artist-name">Azawi <span><img src="./noun_125647_cc-512.webp" alt=""></span></p>
+                    <p class="views-area">2M views . 6months ago</p>
+                </div>
+            </div>
+            <!-- SECOND VIDEO AREA -->
+            <div class="video-area">
+                <div class="sub-video-container">
+                    <img src="./secnd.webp" alt="">
+                    <p>03:45</p>
+                </div>
+                <div class="text-container">
+                    <p class="text-title">
+                        Azawi - Slow Dancing <br>(Official Music Video)
+                    </p>
+                    <p class="artist-name">Azawi <span><img src="./noun_125647_cc-512.web" alt=""></span></p>
+                    <p class="views-area">2M views . 6months ago</p>
+                </div>
+            </div>
+            <!-- THIRD VIDEO AREA -->
+            <div class="video-area">
+                <div class="sub-video-container">
+                    <img src="./3rd.webp" alt="">
+                    <p>03:45</p>
+                </div>
+                <div class="text-container">
+                    <p class="text-title">
+                        Azawi - Slow Dancing <br>(Official Music Video)
+                    </p>
+                    <p class="artist-name">Azawi <span><img src="./noun_125647_cc-512.webp" alt=""></span></p>
+                    <p class="views-area">2M views . 6months ago</p>
+                </div>
+            </div>
+            <!-- FORTH VIDEO AREA -->
+            <div class="video-area">
+                <div class="sub-video-container">
+                    <img src="./coding.webp" alt="">
+                    <p>03:45</p>
+                </div>
+                <div class="text-container">
+                    <p class="text-title">
+                        Azawi - Slow Dancing <br>(Official Music Video)
+                    </p>
+                    <p class="artist-name">Azawi <span><img src="./noun_125647_cc-512.webp" alt=""></span></p>
+                    <p class="views-area">2M views . 6months ago</p>
+                </div>
+            </div>
+
+            <!-- FIFTH VIDEO AREA -->
+            <div class="video-area">
+                <div class="sub-video-container">
+                    <img src="./hqdefault.webp" alt="">
+                    <p>03:45</p>
+                </div>
+                <div class="text-container">
+                    <p class="text-title">
+                        Azawi - Slow Dancing <br>(Official Music Video)
+                    </p>
+                    <p class="artist-name">Azawi <span><img src="./noun_125647_cc-512.webp" alt=""></span></p>
+                    <p class="views-area">2M views . 6months ago</p>
+                </div>
+            </div>
+        </div>
```

Comment:
- [ ] I have noticed that you have wrapped the all the tags in `<section>` tag which is a very good practice. Following the `semantic HTML`, I would suggest to wrap the parent `<div>` tag for specific section inside the `<section>` tag. This is because it helps to improve your website's SEO.
For project preceeding, you may need to implement this change suggested.